recipe/meta.yaml

```
@@ -1,17 +1,17 @@
 # Note that this feedstock must be paired with bumping the version of
 # `arrow-cpp-feedstock` and the SHA-256 hashes should match between the packages.
-{% set version = "17.0.0" %}
+{% set version = "19.0.0" %}
 
 package:
   name: pyarrow
   version: {{ version }}
 
 source:
   url: https://archive.apache.org/dist/arrow/arrow-{{ version }}/apache-arrow-{{ version }}.tar.gz
-  sha256: 9d280d8042e7cf526f8c28d170d93bfab65e50f94569f6a790982a878d8d898d
+  sha256: f89b93f39954740f7184735ff1e1d3b5be2640396febc872c4955274a011f56b
 
 build:
-  number: 1
+  number: 0
   skip: true  # [py<38]
```

https://github.com/apache/arrow/blob/apache-arrow-19.0.0/python/pyproject.toml#L37

```suggestion
  skip: true  # [py<39]
```